src/components/TaskBoard/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import AddTaskCard from "components/TaskCard/AddTaskCard.jsx";
+import "components/TaskBoard/index.scss";
+import TaskList from "components/TaskList";
+
+const TaskBoard = () => {
+  const [showCreateCard, setShowCreateCard] = useState(false);
+  const tasks = useSelector((state) => state.todo);
+
+  function handleCreateTask() {
+    setShowCreateCard(!showCreateCard);
```

Comment:
state updates are asynchronous, calling `setShowCreateCard(!showCreateCard)` multiple times in a row may not toggle the state as expected.